translate/src/modules/machinery/components/source/dropdown.css

```diff
@@ -0,0 +1,49 @@
+.google-translation {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+}
+
+.dropdown-toggle {
+  background: transparent;
+  border: none;
+  cursor: pointer;
+  padding: 0;
+  margin-left: 5px;
+  color: var(--status-translated);
+}
+
+.selected-option {
+  margin-left: 10px;
+  margin-right: 10px;
+  color: var(--status-translated);
+}
+
+.dropdown-menu {
+  display: none;
+  position: absolute;
+  top: 100%;
+  left: 0;
+  display: none;
+  position: absolute;
+  left: 0;
+  background-color: var(--toggle-color-1);
+  color: var(--status-translated);
+  min-width: 160px;
+  box-shadow: 0px 8px 16px 0px var(--toggle-color-1);
+  z-index: 1;
+  border-radius: 4px;
+  overflow: hidden;
+}
+
+.dropdown-menu li {
```

Comment:
The CSS rules selector is not specific enough, so it gets overridden. Prepending it with `.machinery .translation > header ul li` should make the menu items vertical. I'd also move the CSS rules to `MachineryTranslation.css`, which will make it more obvious how specific the selectors need to be.